com.unity.shadergraph/Editor/ShaderFoundry/Linker/Properties/PropertyDeclarations.cs

```diff
@@ -9,10 +9,25 @@ static class PropertyDeclarations
         {
             {"bool", HandleBoolType },
             {"int", HandleIntegerType },
+            {"int1", HandleIntegerType },
```

Comment:
Don't we need bool vector types here?

Reply:
We decided no boolN: https://unity.slack.com/archives/C028KHBJG3G/p1651854215902519?thread_ts=1651853526.697259&cid=C028KHBJG3G